libqtile/backend/base.py

```diff
@@ -680,6 +691,10 @@ def _reset_surface(self):
         )
         self.ctx = self.new_ctx()
 
+    def _create_last_surface(self):
+        """Creates a separate RecordingSurface for mirrors to access."""
+        self.last_surface = cairocffi.RecordingSurface(cairocffi.CONTENT_COLOR_ALPHA, None)
+
```

Comment:
Why is this called the last surface?

Reply:
It's the last drawn surface. Once the drawer is drawn, the recording surface is cleared. The idea is to keep a copy of that RecordingSurface so mirrors can access it.